webapp/channels/src/components/youtube_video/youtube_video.test.tsx

```diff
@@ -91,4 +95,50 @@ describe('YoutubeVideo', () => {
 
         expect(wrapper.find(ExternalImage).prop('src')).toEqual('linkUrl');
     });
+
+    test('should match init snapshot (Shorts)', () => {
+        const store = mockStore(initialState);
+        const props = {
+            ...baseProps,
+            link: 'https://www.youtube.com/shorts/2oa5WCUpwD8',
+        };
+        const wrapper = mount(
+            <IntlProvider locale='en'>
+                <Provider store={store}>
+                    <YoutubeVideo {...props}/>
+                </Provider>
+            </IntlProvider>,
+        );
+        expect(wrapper).toMatchSnapshot();
+        expect(wrapper.find(ExternalImage).prop('src')).toEqual('linkForThumbnail');
```

Comment:
nit: separate this test in two different tests, one only for the snapshot match and the other one that makes sure that the elements you expect to be are actually there.